node/test/hyperbahn/unadvertise.js

```diff
@@ -75,8 +74,7 @@ function runTests(HyperbahnCluster) {
     });
 
     HyperbahnCluster.test('advertise, unadvertise and forward', {
-        size: 5,
-        servicePurgePeriod: 50
```

Comment:
Please keep this test and create a new one in this file for unadvertise endpoint.